### PR TITLE
Switttch branch

### PR DESCRIPTION
**Pull request by Switttch**

### CODE REVIEW

File: Py_04_02_arrays_begin.py [hunk 1]
```diff
@@ -10,6 +10,6 @@
 df2.to_excel(writer,'Sheet1')
 writer.save()
 get_ipython().magic('pwd')
-df3=pd.DataFrame(df1, columns=['whs', 'div', 'yr', 'mnth', 'salesvalue', 'TAC'])
-df3
```
Comment (Switttch) [inline Py_04_02_arrays_begin.py]: 'SPACE HERE'

File: 02_01.py [hunk 2]
```diff
@@ -19,10 +13,8 @@
 from matplotlib import rcParams
 import seaborn as sb
 
-
 # In[3]:
 
-
```
Comment (Switttch) [inline 02_01.py]: DELETE EMPTY LINE

Comment (Swiittch) [reply 02_01.py]: DON't

File: Py_04_02_arrays_begin.py (hunk 1 above)
Comment (Swiittch) [reply Py_04_02_arrays_begin.py]: Add more colour

**Comment by Switttch** [comment]
FUK IT

**Comment by Switttch** [comment]
CHECK3 _ again

**Pull request merged**